gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -239,42 +243,160 @@ if CLIENT then
 		end
 	end
 
 	-- mousebuttons are enough for most weapons
 	local default_key_params = {
-		primaryfire = Key("+attack", "LEFT MOUSE"),
-		secondaryfire = Key("+attack2", "RIGHT MOUSE"),
+		primaryfire = Key("+attack", "MOUSE1"),
+		secondaryfire = Key("+attack2", "MOUSE2"),
 		usekey = Key("+use", "USE")
 	}
 
+	---
+	-- Adds a help text for the weapon to the HUD.
+	-- TTT legacy function.
+	-- @param string|nil primary_text first line of the help text
+	-- @param string|nil secondary_text second line of the help text
+	-- @param[default=false] bool translate should the text get translated
+	-- @param[opt] table extra_params parameters for @{Lang.GetParamTranslation}
+	-- @realm client
 	function SWEP:AddHUDHelp(primary_text, secondary_text, translate, extra_params)
-		extra_params = extra_params or {}
+		local primary = primary_text
+		local secondary = secondary_text
 
-		self.HUDHelp = {
-			primary = primary_text,
-			secondary = secondary_text,
-			translatable = translate,
+		if translate then
+			extra_params = extra_params or {}
 			translate_params = table.Merge(extra_params, default_key_params)
-		}
+			primary = primary and GetPTranslation(primary, translate_params)
+			secondary = secondary and GetPTranslation(secondary, translate_params)
+		end
+
+		--find mouse keys in the texts to add respective icons
+		primary_key = primary and string.find(primary, "MOUSE1") and Key("+attack", "MOUSE1") or nil
+		secondary_key = secondary and string.find(secondary, "MOUSE2") and Key("+attack2", "MOUSE2") or nil
+
+		self:AddTTT2HUDHelp()
+
+		if primary then
+			self:AddHUDHelpLine(primary, primary_key)
+		end
+
+		if secondary then
+			self:AddHUDHelpLine(secondary, secondary_key)
+		end
+	end
+
+	---
+	-- Adds a help text for the weapon to the HUD.
+	-- @param string|nil primary_text description for primaryfire
+	-- @param string|nil secondary_text description for secondaryfire
+	-- @realm client
+	function SWEP:AddTTT2HUDHelp(primary, secondary)
+		self.HUDHelp = {}
+		self.HUDHelp.additional_lines = {}
+		self.HUDHelp.max_length = 0
+
+		if primary then
+			self:AddHUDHelpLine(primary, Key("+attack", "MOUSE1"))
+		end
+
+		if secondary then
+			self:AddHUDHelpLine(secondary, Key("+attack2", "MOUSE2"))
+		end
+	end
+
+	---
+	-- Adds an additional line to the help text.
+	-- @{SWEP:AddTTT2HUDHelp} needs to be called first
+	-- @param string text text to be displayed on the line
+	-- @param Material|string|nil icon_or_key icon or description for the concerning key
+	-- @realm client
+	function SWEP:AddHUDHelpLine(text, icon_or_key)
+		if not self.HUDHelp then return end
+
+		--replace MOUSE1/MOUSE2 strings with respective icons
+		if isstring(icon_or_key) then
+			if icon_or_key == "MOUSE1" then
+				icon_or_key = icon_help_primary
+			elseif icon_or_key == "MOUSE2" then
+				icon_or_key = icon_help_secondary
+			end
+		end
+		icon_or_key = isstring(icon_or_key) and icon_or_key == "MOUSE2" and icon_help_secondary or icon_or_key
```

Comment:
Yea the suggestions didn't supported multiline, so this line have to be removed for sure :D